src/components/AudioPlayer/AudioPlayer.tsx

```diff
@@ -161,12 +162,12 @@ const AudioPlayer = () => {
         {/* The div below serves as placeholder for a right section, as well as for centering the slider */}
         <div className={styles.rightSection}>
           {visibility === Visibility.Expanded && (
-            <Button shape={ButtonShape.Circle} variant={ButtonVariant.Ghost}>
+            <Button tooltip="minimize" shape={ButtonShape.Circle} variant={ButtonVariant.Ghost}>
```

Comment:
I think all tooltips should have the values capitalized e.g. `minimize`-> `Minimize`